roles/openshift_docker_registry/tasks/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+# TODO: Centralized openshift_config openshift_facts??
+# TODO: anchor this on an infrastructure node, default project
+- name: Query for OpenShift Docker Registry
+  command: openshift admin registry --credentials={{ openshift_config }}
+  register: registry_results
+  changed_when: False
```

Comment:
Is there something we can query in the output to know if the router creation was successful or if it already existed?  If so then you can do: `changed_when: "'<changed string>' in registry_results.stdout"` or `changed_when: "'<already exists string>' not in registry_results.stdout"`, or if we can rely on the return code, you can compare against registry_results.rc in `changed_when:`